src/components/dialog/dialog.js

```diff
@@ -34,9 +34,9 @@ define(['dialogHelper', 'dom', 'layoutManager', 'scrollHelper', 'globalize', 're
             dlg.classList.add('dialog-fullscreen-lowres');
         }
 
-        //dlg.querySelector('.btnCancel').addEventListener('click', function (e) {
-        //    dialogHelper.close(dlg);
-        //});
+        dlg.querySelector('.btnCancel').addEventListener('click', function (e) {
```

Comment:
Shouldn't we remove ``e`` variables when they're not used everywhere as well?

Reply:
Yes, but that's for another pass of ESLint :)

For this, I mainly used the 2 rules I added and found some other stuff along the way.

Reply:
There is no `btnCancel` on dialog `Delete media` (careful with that :wink: ).

Reply:
I added some logic to the Metadata editor dialog when I reworked it. I figured it's better to handle that at the dialog level, since I'd like to go over the other dialogs to rework them a bit too.

Reply:
At current moment, dialog is closed this way:
https://github.com/jellyfin/jellyfin-web/blob/bc779ea2383c2ac39061145a1319453b9e12a67a/src/components/dialog/dialog.js#L97-L100
https://github.com/jellyfin/jellyfin-web/blob/bc779ea2383c2ac39061145a1319453b9e12a67a/src/components/dialog/dialog.js#L92-L95